packages/panzer/disc-fe/src/Panzer_PointValues2_impl.hpp

```diff
@@ -109,13 +82,27 @@ namespace panzer {
     point_coords = af.template buildStaticArray<Scalar,Cell,IP,Dim>("point_coords",num_cells, num_points, num_space_dim);
   }
 
-  template <typename Scalar,
-            template <typename DataT,
-               typename Tag0, typename Tag1, typename Tag2,
-               typename Tag3, typename Tag4, typename Tag5,
-               typename Tag6, typename Tag7> class Array >
+  template <typename Scalar>
+  void PointValues2<Scalar>::
+  evaluateValues()
+  {
+    if (point_rule->isSide()) {
+       TEUCHOS_ASSERT(false); // not implemented!!!!
+    }
+    
+    Intrepid2::CellTools<PHX::exec_space> cell_tools;
+    
+    cell_tools.setJacobian(jac.get_view(), coords_ref.get_view(), node_coordinates.get_view(), *(point_rule->topology));
+    cell_tools.setJacobianInv(jac_inv.get_view(), jac.get_view());
+    cell_tools.setJacobianDet(jac_det.get_view(), jac.get_view());
+    
+    // IP coordinates
+    cell_tools.mapToPhysicalFrame(point_coords.get_view(), coords_ref.get_view(), node_coordinates.get_view(), *(point_rule->topology));
+  }
+
+  template <typename Scalar>
   template <typename CoordinateArray>
-  void PointValues2<Scalar,Array>::
+  void PointValues2<Scalar>::
```

Comment:
Me to. This really simplifies things. Though the build times are still really long! :(